webpack.common.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+
+module.exports = {
+    entry: {
+        canvas: './assets/js/src/canvas.js',
+        sidebar: './assets/js/src/sidebar.js',
+        frontend: './assets/js/src/frontend.js',
+        admin: './assets/js/src/admin.js',
+        tinymce: './assets/js/src/tinymce.js',
+    },
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                exclude: /node_modules/,
+                use: {
+                    loader: "babel-loader",
```

Comment:
Does this need a Babel config?

Reply:
if you pull the PR you see the minified JS files aren't different which leads me to believe that I don't, any rules we can put inside this config

Reply:
The minified sidebar JS committed here is different in that it now contains `const`s and arrow functions. If we use any more ES6+ syntax/functions, those won't be transpiled either.